pvlib/transformer.py

```diff
@@ -0,0 +1,126 @@
+"""
+This module contains functions for transformer modeling.
+
+Transformer models calculate AC power output and losses at a given input power.
+"""
+
+import numpy as np
+
+
+def simple_efficiency(
+        input_power, no_load_loss, load_loss, transformer_rating
+):
+    r'''
+    Calculate the energy at the output terminal of the transformer
+    after taking into account efficiency using a simple calculation.
+
+    The equation used in this function can be derived from [1]_.
+
```

Comment:
```suggestion

    For a zero input power, the output power will be negative.
    This means the transformer will consume energy from the grid at night if
    it stays connected (due to the parallel impedance in the equivalent
    circuit).
    If the input power is negative, the output power will be even more
    negative; so the model can be used bidirectionally when drawing
    energy from the grid.
```

I think it's nice to clarify this kind of behaviour in the docstring. Consider this an optional suggestion.